megamek/src/megamek/common/Aero.java

```diff
@@ -4385,4 +4392,82 @@ public void updateSensorOptions() {
             }
         }
     }
+    
+    /**
+     * @return Returns the autoEject.
+     */
+    public boolean isAutoEject() {
+        if (game.getOptions().booleanOption(OptionsConstants.ADVANCED_STRATOPS_QUIRKS)) {
+            return !hasQuirk(OptionsConstants.QUIRK_NEG_NO_EJECT);
+        }
+        return true;
+    }
+
+    /**
+     * @param autoEject
+     *            The autoEject to set.
+     */
+    public void setAutoEject(boolean autoEject) {
+        this.autoEject = autoEject;
+    }
+
+    /**
+     * @return Conditional Ejection Ammo
+     */
+    public boolean isCondEjectAmmo() {
+        return condEjectAmmo;
+    }
+
+    /**
+     * @param condEjectAmmo
+     *            The condEjectAmmo to set.
+     */
+    public void setCondEjectAmmo(boolean condEjectAmmo) {
+        this.condEjectAmmo = condEjectAmmo;
+    }
+
+    /**
+     * @return Conditional Ejection Fuel Explosion
+     */
+    public boolean isCondEjectFuel() {
+        return condEjectFuel;
+    }
+
+    /**
+     * @param condEjectFuel
+     *            The condEjectFuel to set.
+     */
+    public void setCondEjectFuel(boolean condEjectFuel) {
+        this.condEjectFuel = condEjectFuel;
+    }
+
+    /**
+     * @return Conditional Ejection SIDest
+     */
+    public boolean isCondEjectSIDest() {
+        return condEjectSIDest;
+    }
+
+    /**
+     * @param condEjectSIDest
+     *            The condEjectSIDest to set.
+     */
+    public void setCondEjectSIDest(boolean condEjectSIDest) {
+        this.condEjectSIDest = condEjectSIDest;
+    }
+    
+    /**
+     * @return Conditional Ejection CTDest (LAM only)
+     */
+    public boolean isCondEjectCTDest() {
+        return false;
+    }
+
+    /**
+     * @param condEjectCTDest
+     *            The condEjectCTDest to set.
+     */
+    public void setCondEjectCTDest(boolean condEjectCTDest) {
+        this.condEjectCTDest = condEjectCTDest;
+    }
```

Comment:
These methods are never used and don't apply to Aero.

Reply:
As above, they come from IAero because they apply to LAMs.

Reply:
They don't belong in IAero because they apply only to mechs.

Reply:
Fixed